pkg/cli/cliflags/flags.go

```diff
@@ -488,17 +488,31 @@ a public network without combining it with --listen-addr.`,
 The path to the directory containing SSL certificates and keys.
 <PRE>
 
-Cockroach looks for certificates and keys inside the directory using the
+Cockroach looks for required certificates and keys inside the directory using the
 following naming scheme:
 
-  - CA certificate and key: ca.crt, ca.key
-  - Server certificate and key: node.crt, node.key
-  - Client certificate and key: client.<user>.crt, client.<user>.key
+  - ca.crt:                               CA certificate. Used to verify all certificates.
+  - node.crt, node.key:                   Server certificate and key
+  - client.<user>.crt, client.<user>.key: Client certificate and key
+
+All certificates must be signed by ca.crt.
+node.crt must be a dual-purpose client/server certificate, contain "CN=node", and list all used
```

Comment:
Not sure what `CN=Node` means..would the user know what that means? 
Also, can we add guidance about how to generate and use these certs?

Reply:
This is `CommonName`. The CLI help is meant to be a quick description of what the available options are, not a full description of how to use it. For that, they should refer to the docs.

Reply:
👍

Reply:
On further thought, I am not sure if we should have this sentence in the CLI help. It is not complete usage guidance but kinda is - seems like a halfway measure. And it raises more questions than it answers. IMO, we either give complete guidance about all parameters or not mention them here at all and ask them to refer to docs for guidance.